### PR TITLE
SU DGES

### PR DESCRIPTION
**Pull request by OGMalan**
CSL style for Stellenbosch University Department of Geography and Environmental Studies

### CODE REVIEW

File: stellenbosch-university-department-of-geography-and-environmental-studies-reference-style [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+      <name>Ockert Malan</name>
+      <email>19475446@sun.ac.za</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>A CSL style in accordance with SU DGES reference guidelines</summary>
+    <published>01-06-2017</published>
+    <updated>2017-07-07T07:58:22+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <link href="https://stellenbosch-my.sharepoint.com/personal/19475446_sun_ac_za/_layouts/15/guestaccess.aspx?docid=0044a91e96c94404ebe0c42d85021a6c6&amp;authkey=ATwYPE-qyPqiTWUmGI_obXk"/>
```
Comment (rmzelle) [inline stellenbosch-university-department-of-geography-and-environmental-studies-reference-style]: @OGMalan, this documentation link doesn't work for me. Do you know if there is a public copy of the guidelines available. Or, alternatively, could you add the file as an attachment to the pull request?

Comment (OGMalan) [inline stellenbosch-university-department-of-geography-and-environmental-studies-reference-style]: The documentation in question is more a user and setup guide for Mendeley itself.

I do however have the departmental reference guidelines that I could attach?

Additionally, I am fairly inexperienced in XML/CSL and composed this style using the visual editor, so I would greatly appreciate an overview of all the issues that it is.

Comment (rmzelle) [inline stellenbosch-university-department-of-geography-and-environmental-studies-reference-style]: > I do however have the departmental reference guidelines that I could attach?

Yes, if this style is based on those, that's exactly what we're looking for.

Comment (rmzelle) [inline stellenbosch-university-department-of-geography-and-environmental-studies-reference-style]: @OGMalan, if you can attach the departmental reference guidelines here I can help you get this style into the repository.

**Comment by rmzelle** [comment]
@OGMalan, I'm closing this for now because of inactivity. If you can share the reference guidelines here we can just reopen this pull request.

**Pull request closed without merging**